test/Dialect/FIRRTL/infer-widths-errors.mlir

```diff
@@ -159,24 +159,26 @@ firrtl.circuit "Foo" {
 
 // -----
 // https://github.com/llvm/circt/issues/5002
-
-firrtl.circuit "Issue5002" {
-  // expected-error @below {{uninferred width: port "ref" cannot satisfy all width requirements}}
-  firrtl.module private @InRef(in %ref : !firrtl.rwprobe<uint>) { }
-  firrtl.module @Issue5002(in %x : !firrtl.uint<1>, in %y : !firrtl.uint<2>) {
-    %w1, %w1_ref = firrtl.wire forceable : !firrtl.uint, !firrtl.rwprobe<uint>
-    %w2, %w2_ref = firrtl.wire forceable : !firrtl.uint, !firrtl.rwprobe<uint>
-    firrtl.connect %w1, %x : !firrtl.uint, !firrtl.uint<1>
-    firrtl.connect %w2, %y : !firrtl.uint, !firrtl.uint<2>
-
-    %inst1_ref = firrtl.instance inst1 @InRef(in ref: !firrtl.rwprobe<uint>)
-    %inst2_ref = firrtl.instance inst2 @InRef(in ref: !firrtl.rwprobe<uint>)
-    firrtl.ref.define %inst1_ref, %w1_ref : !firrtl.rwprobe<uint>
-    // expected-note @below {{width is constrained to be at most 1 here:}}
-    // expected-note @below {{width is constrained to be at least 2 here:}}
-    firrtl.ref.define %inst2_ref, %w2_ref : !firrtl.rwprobe<uint>
-  }
-}
+// This test is disabled as input probes are now illegal.
+// Similar scenario is now possible with instance_choice.
+
+// firrtl.circuit "Issue5002" {
+//   // expected-error-DISABLED @below {{uninferred width: port "ref" cannot satisfy all width requirements}}
+//   firrtl.module private @InRef(in %ref : !firrtl.rwprobe<uint>) { }
+//   firrtl.module @Issue5002(in %x : !firrtl.uint<1>, in %y : !firrtl.uint<2>) {
+//     %w1, %w1_ref = firrtl.wire forceable : !firrtl.uint, !firrtl.rwprobe<uint>
+//     %w2, %w2_ref = firrtl.wire forceable : !firrtl.uint, !firrtl.rwprobe<uint>
+//     firrtl.connect %w1, %x : !firrtl.uint, !firrtl.uint<1>
+//     firrtl.connect %w2, %y : !firrtl.uint, !firrtl.uint<2>
+// 
+//     %inst1_ref = firrtl.instance inst1 @InRef(in ref: !firrtl.rwprobe<uint>)
+//     %inst2_ref = firrtl.instance inst2 @InRef(in ref: !firrtl.rwprobe<uint>)
+//     firrtl.ref.define %inst1_ref, %w1_ref : !firrtl.rwprobe<uint>
+//     // expected-note-DISABLED @below {{width is constrained to be at most 1 here:}}
+//     // expected-note-DISABLED @below {{width is constrained to be at least 2 here:}}
+//     firrtl.ref.define %inst2_ref, %w2_ref : !firrtl.rwprobe<uint>
+//   }
+// }
```

Comment:
Deleting is preferable to commenting out.

Reply:
Yes, just left it here as a sore thumb so I wouldn't forget to either rework or delete it.   But 100% don't plan to commit it this way :+1:.

Reply:
Will file an issue about it.

Reply:
Dropping.  Was going to file an issue -- I like having the "don't infer probe chain to 'max' width ever" under test but not sure how to make that reachable now, depending on instance choice rules this may not be possible (?).